src/lib/device/journal_vdev.hpp

```diff
@@ -63,7 +63,7 @@ class JournalVirtualDev : public VirtualDev {
 
         off_t m_data_start_offset{0};                   // Start offset of where actual data begin for this vdev
         std::atomic< uint64_t > m_write_sz_in_total{0}; //  Size will be decreased by truncate and increased by append;
-        bool m_truncate_done{true};
+        bool m_truncate_done{false};
         uint64_t m_reserved_sz{0};                       // write size within chunk, used to check chunk boundary;
         std::vector< shared< Chunk > > m_journal_chunks; // Chunks part of this journal in order.
         uint64_t m_total_size{0};                        // Total size of all chunks.
```

Comment:
at line: 71, m_end_offset is actually being updated after init, right? If so, can you update the comment?

Reply:
Yes its updated in init and during recovery time.

Reply:
can you update the comment accordingly at line 71?